service/worker/archiver/archiver.go

```diff
@@ -113,6 +114,7 @@ func handleRequest(ctx workflow.Context, logger log.Logger, metricsClient metric
 		},
 	}
 	actCtx := workflow.WithActivityOptions(ctx, ao)
+	metricsClient.IncCounter(metrics.ArchiverScope, metrics.ArchiverUploadWithRetriesCount)
```

Comment:
technically you can use the `timertype:count` series for this and do not need a new separate emit here.. So then you will have the counts line up with the timer metrics exactly..

Reply:
something like this - fetch name:activity_info_count service:cadence-history deployment:prod-phx2  operation:sessionstats stats_type:count  timertype:count | transformNull 0 | aliasByTags name domain

Reply:
Oh interesting.